### DISCUSSION TITLE
Welcome to ToDoList-web Discussions!

### DISCUSSION THREAD

**Discussion by javohirtohirovich** [javohirtohirovich/ToDoList-web#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abdisamadovich** [comment]
Nima bu ahvol

**Reply by javohirtohirovich** [reply]
it's ToDoList application

**Comment by abdisamadovich** [comment]
![image_2023-08-21_12-45-08](https://github.com/javohirtohirovich/ToDoList-web/assets/85311144/d4e1d357-b4ea-499c-9183-4a9d5777401f)
